projects/rocprofiler-systems/source/lib/core/argparse.cpp

```diff
@@ -368,6 +368,19 @@ add_core_arguments(parser_t& _parser, parser_data& _data)
         _data.processed_environs.emplace("trace");
     }
 
+    if(_data.environ_filter("trace-legacy", _data))
+    {
+        _parser
+            .add_argument({ "-L", "--trace-legacy" },
+                          "Generate a detailed trace (perfetto output)")
+            .max_count(1)
```

Comment:
We probably want to make sure we don't use "trace" and "legacy-trace" at the same time.

```suggestion
            .max_count(1)
            .conflicts({"trace"})
```